forLim.py

```diff
@@ -535,517 +548,553 @@ def run(self):
                 
+                ###Method selection
                 
-                
-                
-                ###################################
-                #  0. traitement tuile par tuile  #
-                ###################################
-                
-                #Get file list
-                path_input = args["Path_input"]
-                files = args["Path_input"].split(";")
-                nfiles = len(files)
-                
-                
-                # Set default values of process bar
-                self.dlg.progressBar.reset()
-                self.dlg.progressBar.setMinimum(1)
-                self.dlg.progressBar.setMaximum(nfiles+9)
-#                if args["Remove_hedges"] or args["Remove_polylines"] or args["Remove_polygons"]:
-#                    self.dlg.progressBar.setMaximum(self.dlg.progressBar.maximum()+1)
-#                if args["Remove_hedges"]:
-#                    self.dlg.progressBar.setMaximum(self.dlg.progressBar.maximum()+1)
-#                if args["Remove_polylines"]:
-#                    self.dlg.progressBar.setMaximum(self.dlg.progressBar.maximum()+1)
-#                if args["Remove_polylines"]:
-#                    self.dlg.progressBar.setMaximum(self.dlg.progressBar.maximum()+1)
-                
-                
-                #Print progress on user window
-                self.dlg.label_printActualProcess.setText("Processing tiles ...")
-                self.dlg.progressBar.setValue(1)
-                QApplication.processEvents() 
-                
-                
-                #Create tiles directory
-                args["Path_output_tiles"] = os.path.join(args["Path_output"],"tiles")
-                os.mkdir(args["Path_output_tiles"])
-                
-                
-                #Process on each tile
-                for f in enumerate(files):
-                    self.dlg.progressBar.setValue(f[0]+2)
-                    QApplication.processEvents() 
-                    args["Path_input"] = f[1]
-                    Overlap_fct.main(args)
-                
-                
-                #Re-set path_input for metadata
-                args["Path_input"] = path_input
-                
-                
-                #Set last path
-                if nfiles == 1:
-                    lastPath = os.path.join(args["Path_output_tiles"], os.path.basename(args["Path_input"]).split('.tif')[0] + ".shp")
-                
-                
-                
-                ####################################################
-                #  1. assembler les tuiles dans un seul shapefile  #
-                ####################################################
-                
-                #Create merge directory
-                args["Path_output_merge"] = os.path.join(args["Path_output"],"merge")
-                os.mkdir(args["Path_output_merge"])
+                if self.dlg.radio_TR.isChecked():
```

Comment:
From here, the if statement makes the code comparison difficult to read since there's one additional tabulation but the code is almost unchanged except the triangulation part.